content/docs/command-reference/add.md

```diff
@@ -336,91 +339,78 @@ $ tree .dvc/cache
 Only the hash values of the `dir/` directory (with `.dir` file extension) and
 `file2` have been cached.
 
-## Example: Transfer to the cache
+## Example: Caching large data externally {#straight-to-cache}
 
-When you have a large dataset in an external location, you may want to add it to
-the <abbr>project</abbr> without having to copy it into the workspace. Maybe
-your local disk doesn't have enough space, but you have setup an
-[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
-that could handle it.
+Sometimes you may want to add a large dataset currently found in an external
+location, so it becomes local to the project. But what if there's not enough
+disk space to download the data?
 
-The `--out` option lets you add external paths in a way that they are
+The `--out` option lets you add external data in a way that it's
 <abbr>cached</abbr> first, and then
 [linked](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache)
-to a given path inside the <abbr>workspace<abbr>. Let's initialize an example
-DVC project to try this:
-
-```dvc
-$ mkdir example # workspace
-$ cd example
-$ git init
-$ dvc init
-```
+to a given path inside the <abbr>workspace</abbr>. Combined with an
+[external cache](/doc/use-cases/shared-development-server#configure-the-external-shared-cache)
+and the right
+[kind of links](/doc/user-guide/large-dataset-optimization#file-link-types-for-the-dvc-cache),
+this let's you avoid using the local file system completely.
 
-Now we can add a `data.xml` file via HTTP for example, putting it a local path
-in our project:
+For example, we can add a `data.xml` file via HTTP, outputting it to a local
+path in our project:
 
 ```dvc
-$ dvc add https://data.dvc.org/get-started/data.xml -o data.xml
+$ dvc add https://data.dvc.org/get-started/data.xml -o raw/data.xml
 
 $ ls
 data.xml data.xml.dvc
 ```
 
-The resulting `.dvc` file will save the provided local `path` as if the data was
-already in the workspace, while the `md5` hash points to the copy of the data
-that has now been transferred to the <abbr>cache</abbr>. Let's check the
-contents of `data.xml.dvc` in this case:
+The local `data.xml` should be a symlink or hard link to the externally
```

Comment:
hardlink?

Reply:
"Hard link" is much more common, it seems (https://trends.google.com/trends/explore?cat=32&geo=US&q=hardlink,hard%20link).

Reply:
it's about consistency in the docs, and DVC itself names them hardlink, symlink, etc. And I think it's acceptable in a sense that people do use `hardlink` as a term

Reply:
btw, hardlinks most likely don't work in this scenario (people will have their cache on a different volume)

Reply:
> it's about consistency in the docs

It looks like we have a mix currently (was planning to change all to "hard link" 😅). We also have a mix of "symlink" and "soft link"...

> hardlinks most likely don't work in this scenario

OK. Since that's the case, I just removed the mention, and we can leave everything else as is.